kernel/init.c

```diff
@@ -146,8 +149,7 @@ void k_call_stacks_analyze(void) { }
  *
  * @return N/A
  */
-void _bss_zero(void)
-{
+void _bss_zero(void) {
```

Comment:
CHECKPATCH ERROR:
 open brace '{' following function declarations go on the next line